src/main/java/frc/robot/Constants.java

```diff
@@ -15,12 +15,28 @@
 public final class Constants {
     public static final double stickDeadband = 0.1;
 
+
+    /* PID Rotation */ 
+    public static final double ROTATE_KP = 0.02; //0.0222
```

Comment:
These constants seem to only be talking about swerve movements, maybe put them in TeleopSwerve?